apps/strapi-cms/src/api/solution-list-page/content-types/solution-list-page/schema.json

```diff
@@ -31,7 +31,7 @@
           "localized": true
         }
       },
-      "type": "text",
+      "type": "string",
```

Comment:
Why did you replace `text` with `string`? I think that `text` is more appropriate than `string` for a `description` field

Reply:
It was a typo